src/components/BookCard/BookCard.module.css

```diff
@@ -0,0 +1,63 @@
+.bookCard {
+  width: 290px;
+  height: 544px;
+
+  @media (max-width: 450px) {
+    width: 316px;
+    height: 544px;
+  }
+}
+
+.cardContent {
+  width: 290px;
+  height: 212px;
+  
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  @media (max-width: 450px) {
+    width: 316px;
+    padding-left: 0;
+  }
+}
+
+.info {
+  width: 270px;
+  padding-left: 20px;
+
+  @media (max-width: 450px) {
+    width: 316px;
+    padding-left: 0px;
+    margin-top: 32px;
+  }
+}
+
+.author {
+  text-decoration: underline;
```

Comment:
You should add a top margin here for author name, so it matches Figma
<img width="250" alt="Screenshot 2024-10-08 at 15 50 48" src="https://github.com/user-attachments/assets/6b6cc087-001f-4bc5-a58b-a88ff6457bf0">